Netkan/Sources/Curse/CurseFile.cs

```diff
@@ -80,13 +75,11 @@ public string GetFileVersion()
         {
             if (_fileVersion == null)
             {
-                // Matches the last group of numbers letters and dots before the .zip extension, staring at a number or a 'v' and a number
-                Match match = Regex.Match(GetDownloadUrl(), "(v?[0-9][0-9a-z.]*[0-9a-z])[^0-9]*\\.zip");
-                if (match.Groups.Count > 1) _fileVersion = match.Groups[1].Value;
-
-                // The id is unique across all files, and is always incrementing.
-                // This format also assures, that any "real" version precedes them.
-                else _fileVersion = GetCurseIdVersion();
+                Match match = Regex.Match(name, "(v?[0-9][0-9a-z.]*[0-9a-z])[^0-9]*\\.zip");
```

Comment:
Can `name` be changed back to `GetDownloadUrl()`, otherwise the `KIS.netkan` won't work. I tried changing the regex pattern, but couldn't get it work.